### ISSUE TITLE
The performance of DISFA dataset

### ISSUE THREAD

**Issue by tghim** [ZhiwenShao/PyTorch-JAANet#22]
Thank you very much for sharing your excellent work.
It's very helpful to me.
But I have some questions about this work. 

1. When I apply your pretrained model for DISFA, I got exactly the same performance as your paper (F1-score, Accuracy). But when I use your pretrained model for BP4D (part1, part3),  f1-score mean and accuracy mean are 0.527 and 0.915, respectively. Was your pretrained model for DISFA trained after trained by whole BP4D dataset?

2. When I apply your pretrained model for DISFA, mean error and failure rate are 4.85 and 1.00 respectively. Did I miss something?

3. For DISFA, I got local f1 score mean 0.576 and local accuracy mean 0.935. The result of using local feature was not good, did you get the same result for DISFA?

Sorry to bother you, but I'd appreciate it if you could answer my question.

**Comment by ZhiwenShao** [comment]
Thanks for your question. Here are the responses:
1. The DISFA model is trained from the best BP4D model. In our experiments, we use the model trained on the combination of BP4D part1 and part3. We do not use the whold BP4D dataset.
2. The results of our trained DISFA models (3-fold cross-validation) are:
f1_score accuracy mean_error failure_rate
0.577556	0.929593	0.043427	0.000000
0.656207	0.939148	0.044319	0.000099
0.671716	0.950716	0.042331	0.000162
==============================
0.635159666666667	0.939819000000000	0.0433590000000000	8.70000000000000e-05
3. The local f1 score is generally worse than f1 score.

**Comment by tghim** [comment]
Thank you for your answer!
It helps me a lot
